app/js/components/Workflows.jsx

```diff
@@ -12,7 +12,7 @@ import Workflow from './Workflow';
 
 const Workflows = ({ plugin, listing, openWorkflow }) => (
     <div>
-        { [...listing].sort((a, b) => a.name > b.name).map(action => (
```

Comment:
This was just completely redundant, the sort is broken like in the other PR, but the elements are already sorted, so it doesn't matter.